terraform/modules/application/ecs.tf

```diff
@@ -0,0 +1,295 @@
+#tfsec:ignore:aws-ecs-enable-container-insight:TODO CLDC-2542 enable container insights if necessary for logging/monitoring
+resource "aws_ecs_cluster" "main" {
+  #checkov:skip=CKV_AWS_65:TODO CLDC-2542 enable container insights if necessary for logging/monitoring
+  name = "${var.prefix}-ecs-cluster"

```

Comment:
I've upgraded the priority of that ticket and cheekily pulled it into sprint, because it's many hours of work easier to change now than it is once we've got anything we can't destroy (e.g. because dns records are pointing at it).

Yeah, I should have caught it earlier but I do think that our convention should not include the resource type in names.